import-export-cli/integration/apim/apim.go

```diff
@@ -1967,21 +2038,12 @@ func getAdminRestURL(host string, offset int, version string) string {
 	return "https://" + host + ":" + strconv.Itoa(port) + "/api/am/admin/" + version
 }
 
-func getTokenURL(host string, offset int) string {
+func getDevOpsRestURL(host string, offset int, version string) string {
```

Comment:
```suggestion
func getDevOpsRestURL(version, host string, offset int) string {
```
Shall we refactor the method signature by grouping the string arguments together, as shown above, to reduce the number of characters per line?

Reply:
Changed all the get rest url methods with requested change.